modules/platforms/cpp/core/include/ignite/cache/event/cache_entry_event.h

```diff
@@ -30,6 +30,45 @@ namespace ignite
 {
     namespace cache
     {
+        /**
+         * Cache entry event type.
+         */
+        struct CacheEntryEventType
```

Comment:
May be better create enum CacheEntryEventType and move FromInt8 to namespace?

Reply:
I am wrong, in absence of enum classes this leads to namespace pollution, this variant is the best for C++0x

Reply:
Yeah, struct approach is better though not so clear.